### ISSUE TITLE
play.spotify.com doesn't appear to work.

### ISSUE THREAD

**Issue by Mikaela** [david-sabata/web-scrobbler#375]
This extension is said to compatible with Spotify, but nothing happens at Last.FM. I have connected this twice.

I'm not sure if I have something wrong somewhere as the play.spotify.com own scrobbler isn't working either.

**Comment by j3mdamas** [comment]
I am also having this issue or a similar one.
Until recently I was only having troubles when I was listening for a long time, where after some time the scrobbling would stop (even though I listened to more songs). Pausing and restarting the songs would resume scrobbling, but it was only temporary. It was annoying.
But since yesterday-ish, it simply does not scrobble. The "This site is supported for scrobbling" red icon is shown, but it does not scrobble. Clicking the icon and changing to next song does not start scrobbling. Re-authenticating, reinstalling the extension, restarting Chrome do not solve the issue.
I have a "ad/track-blocker" which allows me to listen to Spotify without advertisements. I thought it had something to do with it, so I disabled it and tried the scrobbling, but it continued having the same issue.
I'm stuck here :(
